### PR TITLE
Fixed color mismatch and Added Dark Mode

### PR DESCRIPTION
**Pull request by Manan-Rathi**
1. Have fixed the color mismatch of navbar elements
2. Added a button to switch to Dark mode

Screenshot (73)

Screenshot (72)

### CODE REVIEW

File: _includes/header.html
Comment (hex-plex) [inline _includes/header.html]: Thanks, this is really nice can you do one small addition  the button looks a little outdated 
Reference: 
[https://codepen.io/bheberer/pen/BaNZKmq](https://codepen.io/bheberer/pen/BaNZKmq)

Can you use this button instead, There is no javascript only HTML and CSS so should be easy to do

File: css/main.scss
Comment (hex-plex) [inline css/main.scss:252]: Also iguess you can include the title bar in the same

**Review by hex-plex — CHANGES_REQUESTED** [review]
Let me know if you face any difficulty

**Review by hex-plex — APPROVED** [review]

**Comment by hex-plex** [comment]
I have approved the changes but if you can do make the requested changes as well,
Thanks

File: _includes/header.html
Comment (Manan-Rathi) [reply _includes/header.html]: I went through the resources @hex-plex , but everything from the dimensions of the button to the translation of the checkbox has really high values. I'm trying to fix it.

File: css/main.scss
Comment (Manan-Rathi) [reply css/main.scss:252]: Not fixed the button yet. Does the title bar look fine?
Screenshot from 2021-10-09 11-52-12

File: _includes/header.html
Comment (hex-plex) [reply _includes/header.html]: Yeah those can be scaled down I believe, or use a extra div tag and scale down everything at once

File: css/main.scss
Comment (hex-plex) [reply css/main.scss:252]: Yup the bar looks great now

File: _includes/header.html
Comment (Manan-Rathi) [reply _includes/header.html]: I tried scaling down the button. It works now, but there is a glitch. If the you have dark mode enabled and you navigate to another page, dark mode is retained but the btn's style switches to the sun automatically. I tried several things but am unable to fix it.

Screenshot from 2021-10-09 14-26-37

Screenshot from 2021-10-09 14-26-46

Comment (hex-plex) [reply _includes/header.html]: Can you push the code once i can have a look

Comment (Manan-Rathi) [reply _includes/header.html]: Have pushed the code

File: js/bootstrap.min.js
Comment (hex-plex) [inline js/bootstrap.min.js]: So your implementation of mode Switcher is great, only one thing you need to do is, you are saving the theme in local sessionStorage it is so that you can access it everytime throughout the session

so try making a function which always runs when the page is loaded not only when the button is pressed

```js
function initTheme(){
 /* Fetch the theme from the sesionStorage */
 /* Set the document attribute or just call the modeswitcher itself as this would update the button */
}
```

Add this function using the script tag in the end of the body of the HTML file and you should be done

**Review by hex-plex — CHANGES_REQUESTED** [review]
Do this for syncing the theme

File: js/bootstrap.min.js
Comment (Manan-Rathi) [reply js/bootstrap.min.js]: In the final commit, added a function initTheme to find the theme on reload and set the button accordingly. Please review @hex-plex , it works fine now with a slight delay.

Comment (hex-plex) [reply js/bootstrap.min.js]: Great !! :raised_hands: , you are just once step away from a merge 
- [x] just don't call initTheme on `window.onload` this is the one causing the delay you are facing, just call `initTheme();` at the end of the bootstrap.min.js file, and remove the onload definition line
image
- [x] you are making the changes in _site/js but this files are render on runtime so when you made the changes it changed the appearance when you were running but for me it just clears the changes and uses the previous commit
you have to make the changes in `js/bootstrap.min.js` file

tip make your own js files and don't use bootstrap but this time around you have more than enough work for earning a t-shirt all the best !

Comment (Manan-Rathi) [reply js/bootstrap.min.js]: @hex-plex , I guess this is the final commit! Please review.

**Pull request merged**
